main.nf

```diff
@@ -554,7 +557,7 @@ if(params.pseudo_aligner == 'salmon' && !params.salmon_index){
 
         script:
         """
-        salmon index --threads $task.cpus -t $fasta -i salmon_index
+        salmon index --threads $task.cpus -t $fasta --gencode -i salmon_index
```

Comment:
```suggestion
        salmon index --threads $task.cpus -t $fasta $gencode -i salmon_index
```

Reply:
Safest way is to evaluate the parameter beforehand and add that into the command just in case things change in the future. Default option should run without that parameter in agreement with the defaults for `salmon`.

I agree that we need to document this parameter and add it to all of the relevant places in the pipelines 👍